continuous_integration/environment-3.9.yaml

```diff
@@ -1,60 +0,0 @@
-name: dask-distributed
-channels:
-  - conda-forge
-  - defaults
-  - pytorch
-dependencies:
-  - python=3.9
-  - packaging
-  - pip
-  - asyncssh
-  - bokeh
-  - click
-  - cloudpickle
-  - coverage
-  - cython  # Only tested here; also a dependency of crick
-  - dask  # overridden by git tip below
-  - filesystem-spec
-  - gilknocker
-  - h5py
-  - ipykernel
-  - ipywidgets
-  - jinja2
-  - jupyter-server-proxy
-  - jupyterlab
-  - locket
-  - lz4  # Only tested here
-  - msgpack-python
-  - netcdf4
-  - paramiko
-  - pre-commit
-  - prometheus_client
-  - psutil
-  - pyarrow
-  - pynvml  # Only tested here
-  - pytest
-  - pytest-cov
-  - pytest-faulthandler
-  - pytest-repeat
-  - pytest-rerunfailures
-  - pytest-timeout
-  - python-snappy  # Only tested here
-  - pytorch  # Only tested here
```

Comment:
There are a couple of "only tested here" dependencies. We should add them to another environment, probably 3.10 is the best one